src/Windows/R/Components/Impl/Resources.resx

```diff
@@ -348,6 +348,9 @@
   <data name="ConnectionManager_EditLocalTooltip_Format" xml:space="preserve">
     <value>Edit local connection '{0}'</value>
   </data>
+  <data name="ConnectionManager_EditLocalDockerTooltip_Format" xml:space="preserve">
+    <value>Edit local docker connection '{0}'</value>
```

Comment:
Docker is trademark so probably needs to be capitalized. ex Wikipedia `Because Docker containers are so lightweight...`

Reply:
Ok